woocommerce/woocommerce-android
# Firebase remote config improvements

### Description
As discussed in Slack p1659461675819519-slack-CGPNUU63E, this PR is just an attempt to improve our usage of Firebase RemoteConfig SDK by removing DataStore usages from the repository, and as a consequence remove the duplication of default values that was error-prone.

### Testing instructions
Just confirm that the integration still works as expected, you can follow the test steps of this [PR](https://github.com/woocommerce/woocommerce-android/pull/7028) to do so.
